llama-index-integrations/llms/llama-index-llms-anthropic/tests/test_llms_anthropic.py

```diff
@@ -1,7 +1,23 @@
 from llama_index.core.base.llms.base import BaseLLM
 from llama_index.llms.anthropic import Anthropic
+import os
 
 
 def test_text_inference_embedding_class():
     names_of_base_classes = [b.__name__ for b in Anthropic.__mro__]
     assert BaseLLM.__name__ in names_of_base_classes
+
+
+def test_anthropic_through_vertex_ai():
+    if os.getenv("ANTHROPIC_PROJECT_ID"):
```

Comment:
nit: can we skip the test instead? You would need to replace this `if` with the decorator:
```
@pytest.mark.skipif(os.getenv("ANTHROPIC_PROJECT_ID") is None)
```